Tests/Migration/DataPipelineMigrationAssistantTests.swift

```diff
@@ -12,23 +10,34 @@ class DataPipelineMigrationAssistantTests: UnitTest {
     // that have gone undiscovered in the code when `CustomerIO` passes a request to `DataPipelineImplementation`.
     private let profileStoreMock = ProfileStoreMock()
     private let backgroundQueueMock = QueueMock()
-
-    private var migrationAssistant: DataPipelineMigrationAssistant { diGraph.dataPipelineMigrationAssistant }
+    private let migrationHandler = DataPipelineMigrationActionMock()
     private var queueStorage: QueueStorage { diGraph.queueStorage }
+    public var migrationAssistant: DataPipelineMigrationAssistant {
+        DataPipelineMigrationAssistant(handler: migrationHandler)
+    }
 
     override func setUpDependencies() {
         super.setUpDependencies()
 
-        diGraph.override(value: profileStoreMock, forType: ProfileStore.self)
-        diGraph.override(value: dateUtilStub, forType: DateUtil.self)
-        diGraph.override(value: backgroundQueueMock, forType: Queue.self)
+        DIGraphShared.shared.override(value: profileStoreMock, forType: ProfileStore.self)
+        DIGraphShared.shared.override(value: dateUtilStub, forType: DateUtil.self)
+        DIGraphShared.shared.override(value: backgroundQueueMock, forType: Queue.self)
+    }
```

Comment:
`DIGraphShared.shared` should work though, but I would prefer using local variable i.e. `diGraphShared` for consistency. As this is what we used in all other tests.